bait-files/WEB-INF/appengine-generated/datastore-indexes-auto.xml

```diff
@@ -1,4 +1,4 @@
-<!-- Indices written at Thu, 16 Jul 2020 00:24:36 UTC -->
+<!-- Indices written at Fri, 17 Jul 2020 21:42:13 UTC -->
 
 <datastore-indexes/>
 
```

Comment:
This file as well as local_db.bin don't seem to be necessary to the GitHub repository. They are autogenerated locally for you when you run the project on your computer. It would probably be cleanest to set the repository to ignore the appengine-generated folder so that these local temp files don't appear on GitHub.

To do that, you just need to modify the file called **.gitignore** in the project's root directory. I think adding the following line to the file should work:
bait-files/WEB-INF/appengine-generated/

With that .gitignore file updated, these generated files will automatically stop appearing in all of your commits and PRs =)